build.gradle.kts

```diff
@@ -1,3 +1,13 @@
+import io.gitlab.arturbosch.detekt.Detekt
+import io.gitlab.arturbosch.detekt.DetektCreateBaselineTask
+import io.gitlab.arturbosch.detekt.report.ReportMergeTask
+
+plugins {
```

Comment:
@evant plz take a look at change in this file, we will need to coordinate these changes since you made a lot of build script changes in your PR as well.

Reply:
Created a `social.androiddev.codequality` plugin, could probably move all of this to there.

Reply:
Cool, I don't think the merge conflicts should be too bad regardless of which PR gets merged first, so just wanted to give you a heads up.